nengo_spinnaker/config.py

```diff
@@ -1,55 +1,55 @@
 """Nengo/SpiNNaker specific configuration."""
 import nengo
-from nengo.params import BoolParam, DictParam, NumberParam, Parameter, IntParam
+from nengo.params import BoolParam, DictParam, NumberParam, Parameter
 from rig import place_and_route as par
 
 from nengo_spinnaker.node_io import Ethernet
 from nengo_spinnaker.simulator import Simulator
 
 
+def _set_param(obj, name, T, *args, **kwargs):
+    obj.set_param(name, T(name, *args, **kwargs))
```

Comment:
You might want to do something like:

```python
    try:
        obj.set_param(name, T(name=name, *args, **kwargs))
    except TypeError:
        obj.set_param(name, T(*args, **kwargs))
```

Just to have a fallback if someone is using an older version of nengo.

Again, sorry about this -- we completely didn't realize how much we were breaking with this change...